### ISSUE TITLE
Emit carbon costs as metrics

### ISSUE THREAD

**Issue by asdfgugus** [opencost/opencost#2799]
**Is your feature request related to a problem? Please describe.**
Currently, there is no visualization for the introduced carbon cost feature. AFAIK there will be an integration to the OpenCost UI. To provide the user more flexibility, I would like to create a Grafana dashboard for that. However, there are currently no carbon metrics available. 

**Describe the solution you'd like**
Emit the carbon costs as metrics, so that we can scrape them and build a Grafana dashboard. 

**Describe alternatives you've considered**
none

**Additional context**
none

**Comment by younsl** [comment]
I agree with this issue. I spent a significant amount of time trying to figure out why Carbon Cost data was only exposed through the API (`/model/assets/carbon`) but not displayed anywhere in the OpenCost UI. It would be very helpful to have carbon cost metrics visible in the UI alongside the existing cost data.

**Comment by enriqueav99** [comment]
Hi there! I'd like to work on this, since in my case I scrape the metrics instead of calling the OpenCost API, so I'd like to push this forward.

 There's already a solid piece of work on this from PR #3162 , which adds the needed changes to metrics.go and handlers.go, but it looks like it's stalled, the last activity was back in July 2025. I'd be happy to pick that up and get it across.
